packages/components/src/components/hds/interactive/index.ts

```diff
@@ -12,10 +12,10 @@ export interface HdsInteractiveSignature {
     isHrefExternal?: boolean;
     isRouteExternal?: boolean;
     route?: string;
-    models?: Array<string | number>;
-    model?: string | number;
-    query?: Record<string, string>;
-    'current-when'?: string;
+    models?: unknown[];
```

Comment:
[suggestion] consider adding a comment somewhere that clarify where these types "come from" and why we can't import them directly, so in the future we avoid going through the whole process again ;) 
```suggestion
    models?: unknown[];
```